notebooks/kubeflow_pipelines/cicd/labs/kfp_cicd.ipynb

```diff
@@ -267,7 +267,7 @@
     "|Source| [YOUR FORK]|\n",
```

Comment:
<p>I think we should keep <code>./notebooks/kubeflow_pipelines/cicd/labs/cloudbuild.yaml</code>  otherwise the student exercise cloudbuild.yam won't be used but only the solution, which is not what we want for the exercises. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
@takumiohym  Can you explain the problem you are trying to solve? I am not sure I am fully understanding the issue.

Reply:
@BenoitDherin Thanks for your comment.

The lab is saying `Cloud Build configuration file location` should be specified as `./notebooks/kubeflow_pipelines/cicd/labs/cloudbuild.yaml`, but since the modified file is not pushed to the repository in this lab, Cloud Build just refers to the original [empty yaml file](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/notebooks/kubeflow_pipelines/cicd/labs/cloudbuild.yaml), and it doesn't work.

So the solution should be either 
1. changing the path to `solution` in which all the variables are defined, or 
2. adding steps to push the modified yaml file to the repository.

From the discussion with Daniel above, the option 2 would be preferable.
This can bee the possible implementation of 2.
![image](https://user-images.githubusercontent.com/6895245/139873752-64c8d5e8-255b-4444-bd0c-44ce72f94f67.png)

Reply:
Thanks @takumiohym for the clarification. I got it now. Let's go with solution 2, but instead of having a cell with the code to push the code, let's just add that as a comment letting know the student needs to push the code .